osmosis-1/osmosis-1.assetlist.json

```diff
@@ -656,6 +656,34 @@
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/med.png"
       },
       "coingecko_id": "medibloc"
+    },
+    {
+      "description": "The staking token of Bostrom",
+      "denom_units": [
+        {
+          "denom": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4",
+          "exponent": 0,
+          "aliases": [
+            "BOOT"
+          ]
+        },
+        {
+          "denom": "boot",
+          "exponent": 0
+        }
+      ],
+      "base": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4",
+      "name": "Bostrom",
+      "display": "boot",
+      "symbol": "BOOT",
+      "ibc": {
+        "source_channel": "channel-2",
+        "dst_channel": "channel-95",
+        "source_denom": "boot"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/boot.png"
```

Comment:
Can you change name image : bostrom.png -> boot.png
Thanks

Reply:
Done